.github/workflows/ms.automation.automationaccounts.yml

```diff
@@ -23,8 +23,8 @@ on:
         default: '0.0.1'
 
   push:
-    branches:
-      - main
+    #branches: #TODO
```

Comment:
probably a leftover

Reply:
will fix again when I have tested SA MI approach. Though I see inconsistencies across our repo on this.

Reply:
https://github.com/Azure/ResourceModules/runs/4336227065?check_suite_focus=true#step:4:329

Seems its not working by default. Seems Azure has some inconsistencies on this area.

Reply:
fixed